clients/cloud/java/README.md

```diff
@@ -203,4 +203,9 @@ In the output below, substitute values for `{{ SR_API_KEY }}`, `{{ SR_API_SECRET
     $ curl -u {{ SR_API_KEY }}:{{ SR_API_SECRET }} https://{{ SR_ENDPOINT }}/subjects/test2-value/versions/1
     {"subject":"test2-value","version":1,"id":100001,"schema":"{\"name\":\"io.confluent.examples.clients.cloud.DataRecordAvro\",\"type\":\"record\",\"fields\":[{\"name\":\"count\",\"type\":\"long\"}]}"}
     ```
+8. For schema evolution: you can test schema compatibility between newer schema versions and older schema versions in Confluent Cloud Schema Registry. In the [pom.xml](pom.xml#L189) file, set the subject name (i.e., `test2-value`) and local schema to test (in this example, [DataRecordAvro2a](src/main/resources/avro/io/confluent/examples/clients/cloud/DataRecordAvro2a.avsc) should fail and [DataRecordAvro2b](src/main/resources/avro/io/confluent/examples/clients/cloud/DataRecordAvro2b.avsc) should pass).
+
+   ```
+   $ mvn io.confluent:kafka-schema-registry-maven-plugin:5.5.1:test-compatibility "-DschemaRegistryUrl=https://{{ SR_ENDPOINT }}" "-DschemaRegistryBasicAuthUserInfo={{ SR_API_KEY }}:{{ SR_API_SECRET }}"
```

Comment:
Would a shorter `mvn schema-registry:test-compatibility ...` work instead?

Reply:
@mikebin I did not know about this shorter form, thanks for pointing it out.  Yes, it does work as well.  I can update to this form if you think it's more suitable (versus longer form in which sometimes being more explicit is beneficial?)

Reply:
I'm not sure if there's a preferred way. I lean slightly toward `schema-registry:test-compatibility`, because it's shorter, and more closely matches what's in the docs. But either way is fine with me.